sdk/graphservices/Azure.ResourceManager.GraphServices/tests/Tests/AccountCRUDTests.cs

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Threading.Tasks;
+using Azure.ResourceManager.Resources;
+using Azure.ResourceManager.GraphServices.Tests.Helpers;
+using Azure.ResourceManager.GraphServices.Models;
+using NUnit.Framework;
+using Azure.Core.TestFramework;
+
+namespace Azure.ResourceManager.GraphServices.Tests.Tests
+{
+    [TestFixture]
+    public class AccountCRUDTests : GraphServicesManagementTestBase
+    {
+        public AccountCRUDTests() : base(true)
+        {
+        }
```

Comment:
we should not hard code the `isAsync` to true here.
The test framework always ensures us to have both version of the test cases, sync and async.
```suggestion
        public AccountCRUDTests(bool isAsync) : base(isAsync)
        {
        }
```
This might need a re-recording

Reply:
Also please ensure that every API we are calling in our test cases is the ASYNC version.
The test framework we automatically replace those async alls with their corresponding sync version when `isAsync` is false.